R/derive_param_computed.R

```diff
@@ -241,3 +241,45 @@ derive_derived_param <- function(dataset,
 
   bind_rows(dataset, hori_data)
 }
+
+#' Adds a Parameter Computed from the Analysis Value of Other Parameters
+#'
+#' @description
+#' `r lifecycle::badge("deprecated")`
+#'
+#' This function is deprecated.  Please use `derive_param-computed()` instead.
+#'
+#' @inheritParams derive_param_computed
+#'
+#' @author Stefan Bundfuss
+#'
+#' @return The input dataset with the new parameter added. Note, a variable will only
+#'    be populated in the new parameter rows if it is specified in `by_vars`.
+#'
+#' @keywords derivation bds
+#'
+#' @export
+#'
+derive_derived_param <- function(dataset,
+                                 by_vars,
+                                 parameters,
+                                 analysis_value,
+                                 set_values_to,
+                                 filter = NULL,
+                                 constant_by_vars = NULL,
+                                 constant_parameters = NULL) {
+  ### BEGIN DEPRECATION
+  deprecate_warn("0.8.0", "derive_derived_param()", "derive_param_computed()")
+  derive_param_computed(
+    dataset,
+    by_vars,
+    parameters,
+    analysis_value,
+    set_values_to,
+    filter,
+    constant_by_vars,
+    constant_parameters
+  )
```

Comment:
Those parameters which are passed as symbols (i.e. `anaylsis_value`) or expressions (i.e. `filter`) please don't pass the variables along as is but rather use `!!enquo(<parameter>)`. Also please explicitly name all parameter but `dataset`, e.g. `by_vars = by_vars` rather than just `by_vars`.